### PR TITLE
CVE-2011-1196

### PR DESCRIPTION
**Pull request by dkgramming**

### CODE REVIEW

File: cves/CVE-2011-1196.yml [hunk 1]
```diff
@@ -34,10 +41,16 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
+- :commit:
```
Comment (dkgramming) [inline cves/CVE-2011-1196.yml]: I wasn't able to find the commit hash for this change

File: cves/CVE-2011-1196.yml [hunk 2]
```diff
@@ -15,7 +15,14 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  Heap-based overflow in Google Chrome's implementation of the OGG metafile 
+  format allows remote attackers to cause a denial of service or possibly have
```
Comment (brandoncornel) [inline cves/CVE-2011-1196.yml]: I think that this should be past tense. So allows -> allowed.

File: cves/CVE-2011-1196.yml [hunk 3]
```diff
@@ -77,10 +91,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer: 
-  date: 
+  answer: |
+    The vulnerability was initial found while running the Windows version of
```
Comment (brandoncornel) [inline cves/CVE-2011-1196.yml]: initial should be fixed to initially

File: cves/CVE-2011-1196.yml [hunk 4]
```diff
@@ -178,3 +194,7 @@ mistakes:
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
   answer: 
+    It is not entirely clear what the root cause of the vulnerability is. There
+    is a comment in review 6598004 stating that a stream index is mysteriously
+    becoming invalid. This leeds me to think that there is some shared mutable
```
Comment (brandoncornel) [inline cves/CVE-2011-1196.yml]: Leeds should be leads

File: cves/CVE-2011-1196.yml [hunk 5]
```diff
@@ -15,7 +15,14 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
```
Comment (brandoncornel) [inline cves/CVE-2011-1196.yml]: I really like this CVE! I'll give this 5 points!

**Pull request merged**